src/_locales/en/messages.json

```diff
@@ -0,0 +1,21 @@
+{
+    "extensionDescription": {
+        "message": "Firefox Relay makes it easy to create email aliases that forward to your real inbox. Use it to protect your online accounts from hackers and unwanted messages.",
+        "description": "Description of the extension. DO NOT TRANSLATE \"Firefox Relay\"."
+    },
+    "firstRunP": {
+        "message": "You’ve successfully installed the Firefox Relay add-on. To get started, sign in to Relay with your Firefox account."
+    },
+    "firstRunCollectDataLabel": {
+        "message": "Allow Mozilla to collect information about how you use this extension."
+    },
+    "disableDataCollection": {
+        "message": "Datensammlung deaktivieren"
```

Comment:
This isn't English looking.

Reply:
oops, got my copy/pastes crossed. thanks for being thorough!

Reply:
Fixed this one.